spec/01-lexical-syntax.md

```diff
@@ -553,6 +553,20 @@ but are required to be properly nested.  Therefore, a comment like
 `/* /* */` will be rejected as having an unterminated
 comment.
 
+## Multi-line and trailing commas
+
+If a comma is followed by a line end and then a right parenthesis, square
+bracket or curly brace (`)`, `]`, and `}`, respectively) then such a trailing
+comma is ignored. For example:
```

Comment:
specese is "newline" and parentheses are defined on pg 1. Suggestion to follow after walk to school.